### PR TITLE
Emit NAVUpdated event on borrow, repay, writeoff

### LINKED ISSUE #641: Add `NAVUpdated` events on borrow/repay/write-off

### PR DESCRIPTION
**Pull request by branan**
This changes the NAVUpdated event to have an `exact` parameter, and
adds inexact NAVUpdated events for loan actions which modify the NAV.

Closes #641

### CODE REVIEW

**Review by mustermeiszer — APPROVED** [review]
LGMT!

File: pallets/loans/src/lib.rs [hunk 1]
```diff
@@ -212,8 +212,8 @@ pub mod pallet {
 		/// An amount was repaid for a loan. [pool, loan, amount]
 		Repaid(PoolIdOf<T>, T::LoanId, T::Amount),
 
-		/// The NAV for a pool was updated. [pool, nav]
-		NAVUpdated(PoolIdOf<T>, T::Amount),
+		/// The NAV for a pool was updated. [pool, nav, exact]
+		NAVUpdated(PoolIdOf<T>, T::Amount, bool),
```
[the remark this answers is not in the document]
Comment (branan) [reply pallets/loans/src/lib.rs]: Good idea, done. I'm open to naming suggestions

**Review by NunoAlexandre — APPROVED** [review]
🍯 🐻

File: pallets/loans/src/lib.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (NunoAlexandre) [reply pallets/loans/src/lib.rs]: I like what you went with!

**Review by offerijns — APPROVED** [review]
LGTM! Great suggestion @NunoAlexandre 🔥

**Pull request merged**